app/src/androidTestMock/java/com/example/android/architecture/blueprints/todoapp/tasks/TasksActivityTest.kt

```diff
@@ -104,11 +109,10 @@ class TasksActivityTest {
 
     @Test
     fun editTask() {
-        repository.saveTaskBlocking(Task("TITLE1", "DESCRIPTION"))
+        dataBindingIdlingResource.monitorActivity(composeTestRule.activityRule.scenario)
```

Comment:
What does this do? How come we still need it when you are removing data binding for Compose?

Reply:
This is an end2end test that still uses the Activity and some other Fragments. We'll be able to remove this in the last PR when everything is migrated to Compose, but data binding is still used in some parts of the app at the moment